types/soundmanager2/index.d.ts

```diff
@@ -0,0 +1,191 @@
+// Type definitions for soundmanager2 2.97
+// Project: https://github.com/scottschiller/SoundManager2
+// Definitions by: Elton Lee <https://github.com/elton2048>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.3
+
+declare var soundManager: soundmanager.SoundManager;
+
+declare namespace soundmanager {
+    type ScriptAccess = 'always' | 'sameDomain';
+
+    interface DefaultOptions {
+        autoLoad?: boolean;
+        autoPlay?: boolean;
+        from?: number | null;
+        loops?: number;
+        onid3?(): void | null;
```

Comment:
I'm a little bit confused by the type here. 
Did you mean `onid3?: (() => void) | null;`? This would mean that onid3 could either be null, undefined, or a function that takes no arguments and does not return a value.

Reply:
Yes that's the cases. All the usage of ``` void | null ``` in the file means that case.
Do I need to update the new version to merge the commit then?

Reply:
Yes, you will need to use the version I suggested to properly express the type that you intend.